api/src/test/java/io/grpc/LoadBalancerTest.java

```diff
@@ -127,6 +124,7 @@ public void pickResult_equals() {
 
   @Deprecated
   @Test
+  @SuppressWarnings("deprecation")
```

Comment:
I thought we should have comments on SuppressWarnings to let people know why you are suppressing them and why it is safe.

Reply:
These are tests. And only for getting rid of lint warnings. Whatever, I will just remove them.

Reply:
@ran-su When you add `@Deprecated` to the method on purpose (e.g. these tests will be removed once the deprecated API is removed), it does not need `@SuppressWarnings("deprecation")` anymore.